dev-client/src/components/common/Card.tsx

```diff
@@ -30,18 +30,48 @@ export const CardCloseButton = (
   );
 };
 
+export const CardTriangle = () => {
+  return (
+    <Box
+      width={0}
+      height={0}
+      borderBottomWidth={15}
+      borderBottomColor="white"
+      borderLeftWidth={15}
+      borderLeftColor="transparent"
+      borderRightWidth={15}
+      borderRightColor="transparent"
+      alignSelf="center"
+      position="absolute"
+      top={-14}
```

Comment:
comment: i still feel a bit affronted that this is one of the most reliable ways to make a triangle in 2023